.travis.yml

```diff
@@ -118,6 +146,21 @@ jobs:
     dist: xenial
     env:
     - PACKAGE=pnr/vtr
+  - stage: "EDA Tools - PnR"
+    os: linux
+    dist: xenial
+    env:
+    - PACKAGE=pnr/vtr-gui
+  - stage: "EDA Tools - PnR"
```

Comment:
Can we separate out symbiflow synthesis stuff from the symbiflow-vtr stuff into separate pull requests?